packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -338,6 +338,16 @@ module.exports = async ({
     }
   }
 
+  /**
+   * Loads pdf assets
+   */
+  rules.pdfs = () => {
+    return {
+      use: [loaders.url()],
```

Comment:
Should this be `file()` to have same behaviour as in v1? `url()` will inline assets under 10kB

Reply:
Woops! I believe you are correct. Pushing the changes right now. Thanks for the feedback!